opendj-packages/opendj-docker/bootstrap/setup.sh

```diff
@@ -5,31 +5,61 @@ echo "Setting up default OpenDJ instance"
 
 # If any optional LDIF files are present load them
 
-/opt/opendj/setup --cli -p $PORT --ldapsPort $LDAPS_PORT --enableStartTLS $OPENDJ_SSL_OPTIONS \
-  --baseDN $BASE_DN -h localhost --rootUserDN "$ROOT_USER_DN" --rootUserPassword "$ROOT_PASSWORD" \
-  --acceptLicense --no-prompt  $ADD_BASE_ENTRY #--sampleData 1
+/opt/opendj/setup \
+  --cli \
+  -h localhost \
+  --baseDN $BASE_DN \
+  --ldapPort $PORT \
+  --ldapsPort $LDAPS_PORT \
+  --enableStartTLS $OPENDJ_SSL_OPTIONS \
+  --adminConnectorPort $ADMIN_PORT \
+  --rootUserDN "$ROOT_USER_DN" \
+  --rootUserPassword "$ROOT_PASSWORD" \
+  --acceptLicense \
+  --no-prompt \
+  --noPropertiesFile \
+  --doNotStart \
+  $ADD_BASE_ENTRY #--sampleData 1
+
+# There are multiple types of ldif files.
+# This step makes plain copies.
+# See below for imports via `ldapmodify`.
+if [ -d /opt/opendj/bootstrap/config/schema/ ]; then
```

Comment:
This is the relevant change, I also had to add `--doNotStart` in the command line above and an explicit `start-ds` below, so that OpenDJ will pick up the copied ldif files.